src/react-compat.js

```diff
@@ -134,6 +134,8 @@ if (REACT013) {
   };
 }
 
+const batchedUpdates = require('react/lib/ReactUpdates').batchedUpdates;
```

Comment:
I think we'll likely have to be reactive when it comes to https://github.com/facebook/react/issues/6460. We should definitely try to work with the React team if they want to support libraries like enzyme in some capacity.

Also @koba04 can you put this `require` up at the top with the main `const React = require('react')` statement, since this is not a version dependant import at all.